x/stakeibc/keeper/hooks.go

```diff
@@ -236,11 +237,18 @@ func (k Keeper) GetUndelegatedBalance(chainId string, depositRecords []recordsty
 
 // Returns the total delegated balance that's stored in LSM tokens
 // These are identified by any status besides "DEPOSIT_PENDING"
```

Comment:
Could you please add a comment here explaining why PENDING is the only status that's not counted?

Reply:
done